contracts/interfaces/IDistributor.sol

```diff
@@ -28,13 +28,13 @@ interface IDistributor is IComponent {
     /// @param dest The address set to receive the distribution
     /// @param rTokenDist The distribution of RToken that should go to `dest`
     /// @param rsrDist The distribution of RSR that should go to `dest`
-    event DistributionSet(address dest, uint16 rTokenDist, uint16 rsrDist);
+    event DistributionSet(address indexed dest, uint16 rTokenDist, uint16 rsrDist);
```

Comment:
Very nice. Was going to comment that I think we should add it here, but you already saw that.